packages/artifacts/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@aragon/admin-plugin-artifacts",
```

Comment:
If there is no `@aragon/admin-plugin` package, do we need the `-artifacts` suffix? Minor comment

Reply:
i thought it show be more explicit